content/ui-ux.md

```diff
@@ -14,11 +14,40 @@ After reading this guide, you'll know:
+
+
+**Syntax**
+- Blaze uses an easy-to-learn [Handlebars](http://handlebarsjs.com)-like template syntax, with logic like `{% raw %}{{#if}}{% endraw %}` and `{% raw %}{{#each}}{% endraw %}` interspersed in your HTML files. Template functions and CSS-selector events maps are written in Javascript files.
+- React uses [JSX](https://facebook.github.io/react/docs/jsx-in-depth.html), with which you write your HTML in Javascript. While it doesn't have the logic-view separation most libraries have, it also has the most flexibility. Template functions are TODO
+- Angular uses HTML with [special attribute syntax](https://angular.io/docs/ts/latest/guide/cheatsheet.html) for logic and events. Template functions are TODO
+- React and Angular enforce a better component structure, which makes developing larger apps easier.
 
-In Meteor, we officially support three user interface (UI) rendering libraries, [Blaze](blaze.html), [React](http://react-in-meteor.readthedocs.org/en/latest/) and [Angular](http://www.angular-meteor.com).
+**Community**
+- Blaze has many full-stack Meteor packages on Atmosphere, such as [`useraccounts:core`](https://atmospherejs.com/useraccounts/core) and [`aldeed:autoform`](https://atmospherejs.com/aldeed/autoform).
+- React has 41k stars on Github and 13k npm libraries.
+- Angular has 11k stars on Github and 4k npm libraries.
+
+**Performance**
+ - Render performance varies a lot depending on the situation. All three libraries are very quick at rendering simple apps, but can take a noticeable amount of time with more complex apps.
+ - Angular and React have had more performance optimization work put into them than Blaze and in general will perform better. However, there are some cases when Blaze does better (for instance an `{% raw %}{{#each}}{% endraw %}` over a changing cursor).
+ - [One test](http://info.meteor.com/blog/comparing-performance-of-blaze-react-angular-meteor-and-angular-2-with-meteor) benchmarks Angular 2 as the best, followed by React and Angular 1, followed by Blaze. 
+
+**Mobile**  
+- **Cordova**
+  - All three libraries work fine in a Cordova web view, and you can use mobile CSS libraries like Ionic's CSS with any view library.
+  - The most advanced mobile web framework is [Ionic 2](http://ionicframework.com/docs/v2/), which uses Angular 2.
+  - Ionic 1 uses Angular 1, but there are also [Blaze](http://meteoric-doc.com/) and [React](http://reactionic.github.io/) ports.
+  - If you're using React, their most popular mobile web framework is [Reapp](http://reapp.io/).
```

Comment:
Maybe avoid 'their' because Reapp is an independent project not owned by the React team.